include/build_config/SDL_build_config.h.cmake

```diff
@@ -31,37 +31,31 @@
 /* General platform specific identifiers */
 #include <SDL3/SDL_platform_defines.h>
 
-/* C datatypes */
-/* Define SIZEOF_VOIDP for 64/32 architectures */
-#if defined(__LP64__) || defined(_LP64) || defined(_WIN64)
-#define SIZEOF_VOIDP 8
-#else
-#define SIZEOF_VOIDP 4
-#endif
+#cmakedefine HAVE_GCC_ATOMICS 1
+#cmakedefine HAVE_GCC_SYNC_LOCK_TEST_AND_SET 1
 
-#cmakedefine HAVE_GCC_ATOMICS @HAVE_GCC_ATOMICS@
-#cmakedefine HAVE_GCC_SYNC_LOCK_TEST_AND_SET @HAVE_GCC_SYNC_LOCK_TEST_AND_SET@
+#cmakedefine SDL_DISABLE_ALLOCA 1
 
-#cmakedefine SDL_DISABLE_ALLOCA
+/* Useful headers */
+#cmakedefine HAVE_FLOAT_H 1
+#cmakedefine HAVE_STDARG_H 1
+#cmakedefine HAVE_STDDEF_H 1
+#cmakedefine HAVE_STDINT_H 1
 
 /* Comment this if you want to build without any C library requirements */
 #cmakedefine HAVE_LIBC 1
 #ifdef HAVE_LIBC
 
 /* Useful headers */
 #cmakedefine HAVE_ALLOCA_H 1
-#cmakedefine HAVE_FLOAT_H 1
 #cmakedefine HAVE_ICONV_H 1
 #cmakedefine HAVE_INTTYPES_H 1
+#cmakedefine HAVE_LIBUNWIND_H 1
```

Comment:
This macro can be removed from all `SDL_build_config*` headers. It's only used by SDL_test, which does not use the config header.